rviz_default_plugins/src/rviz_default_plugins/displays/image/image_display.hpp

```diff
@@ -47,6 +47,7 @@
 # include "rviz_common/properties/float_property.hpp"
 # include "rviz_common/properties/int_property.hpp"
 # include "rviz_common/properties/queue_size_property.hpp"
+#include "../../visibility_control.hpp"
```

Comment:
This relative import does not work when these files are automoc'ed on Windows, e.g.:

```
03:48:15          c:\j\workspace\ci_windows\ws\build\rviz_default_plugins\rviz_default_plugins_autogen\wgdjm44luk\../../../../src/ros2/rviz/rviz_default_plugins/src/rviz_default_plugins/displays/image/image_display.hpp(50): fatal error C1083: Cannot open include file: '../../visibility_control.hpp': No such file or directory (compiling source file C:\J\workspace\ci_windows\ws\build\rviz_default_plugins\rviz_default_plugins_autogen\mocs_compilation.cpp) [C:\J\workspace\ci_windows\ws\build\rviz_default_plugins\rviz_default_plugins.vcxproj]
```

I recommend making that header public instead...

Actually, what is the issue you run into when you don't use visibility macros? I would have expected the plugins to not need them...